### PR TITLE
Fixed nginx issues

### PR DESCRIPTION
**Pull request by neelp03**
For some reason, all APIs stopped working. These changes should fix it..

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -53,10 +52,10 @@ http {
...
         }
-	location / {
+        location / {
             proxy_pass http://webserver;
         }
     }
 }
 
-events { }
+events { }
```
Comment (evanugarte) [inline nginx.conf:61]:
```suggestion
events { }

```

File: nginx.conf [hunk 2]
```diff
@@ -6,18 +6,17 @@ http {
     #logging
     access_log  /var/log/nginx/sce.access.log;
 
-    #upstream connection to node servers from npm run server
```
Comment (evanugarte) [inline nginx.conf:9]: paste this somewhere in both `.conf` files for reference:
```nginx
# when we run docker-compose up for the first time, a default
# network is created that all the containers are attached to.
# this default network name is the directory name + "_default".
# As a result, the network name will be "core-v4_default".
# To establish a connection to the containers, we can do
# <container name>.<network name>, and that's what happens below.
```

**Review by evanugarte — APPROVED** [review]
Thank you @aaj-gd for the assist

**Review by imdominicreed — APPROVED** [review]
BING CHILLING 🥶

**Pull request merged**